### ISSUE TITLE
Picks up files in _build

### ISSUE THREAD

**Issue by htgoebel** [bjones1/CodeChat#10]
CodeChat picks up files in `_build` directory. I'd expect it to not pick up these files.

1. Set up a simple sphinx project, which *does not separate* source and build dirs.
2. Add "CodeChat.CodeToRestSphinx" and "" and to these projects sphinx plugins
3. Run `make html`
4. Run `make html` again
5. Run `find _build/html/ -iname \*.js.\*`
   You will get entries like these,  showing the even file in `_build` are processed
      ```
      _build/html/_sources/_build/html/_static/jquery.js.txt
      _build/html/_build/html/_static/searchtools.js.html
      ```

**Comment by bjones1** [comment]
This is [standard Sphinx behavior](https://github.com/sphinx-doc/sphinx/blob/3.x/sphinx/cmd/quickstart.py#L371) -- you must manually exclude the `_build` directory (or whatever it's named) in any Sphinx project.

**Comment by htgoebel** [comment]
>  assume you've already seen the provided Sphinx templates in https://github.com/bjones1/CodeChat/tree/master/CodeChat/template? 

No, I just took as abasis what I learned from the the pyinstaller hooksample :-) I had an already existing project, so a complete new template was not what I need. So maybe this projects conf.py was wrong already, but since _build did not include anx .rst files, this go unnoticed.

Regarding this template: While it is complete, and because it is complete, it hinders finding the relevant point.
